lib/sqlalchemy/orm/__init__.py

```diff
@@ -177,17 +177,27 @@ def deferred(*columns, **kw):
     return ColumnProperty(deferred=True, *columns, **kw)
 
 
-def query_expression():
+def query_expression(default_expr=_sql.null()):
     """Indicate an attribute that populates from a query-time SQL expression.
 
+    :param default_expr:
+        SQL clause object. The default query expression if
+        not assigned later by `with_expression`. E.g.::
+            from sqlalchemy.sql import literal
```

Comment:
blank space is needed after the `::` I think

Reply:
blank line added